docs/guides/mlp_tutorials/llm-inference.md

```diff
@@ -0,0 +1,254 @@
+  warnings.warn(
+Gemma's activation function should be approximate GeLU and not exact GeLU.
+Changing the activation function to `gelu_pytorch_tanh`.if you want to use the legacy `gelu`, edit the `model.config` to set `hidden_activation=gelu`   instead of `hidden_act`. See https://github.com/huggingface/transformers/pull/29402 for more details.
+Loading checkpoint shards: 100%|██████████| 4/4 [00:03<00:00,  1.13it/s]
+/capstor/scratch/cscs/user/gemma-inference/gemma-venv/lib/python3.10/site-packages/huggingface_hub/file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
+  warnings.warn(
+<bos>Write me a poem about the Swiss Alps.
+
+In the heart of Switzerland, where towering peaks touch sky,
+Lies a playground of beauty, beneath the watchful eye.
+The Swiss Alps, a majestic force,
+A symphony of granite, snow, and force.
+
+Snow-laden peaks pierce the heavens above,
+Their glaciers whisper secrets of ancient love.
+Emerald valleys bloom with flowers,
+A tapestry of colors, a breathtaking sight.
+
+Hiking trails wind through meadows and woods,
+Where waterfalls cascade, a silent song unfolds.
+The crystal clear lakes reflect the sky above,
+A mirror of dreams, a place of peace and love.
+
+The Swiss Alps, a treasure to behold,
+A land of wonder, a story untold.
+From towering peaks to shimmering shores,
+They inspire awe, forevermore.<eos>
+```
+
+Congrats! You've run Google Gemma-7B inference on four GH200 chips simultaneously. Move on to the next tutorial or try the challenge.
+
+### Challenge
+
+Using the same approach as in the latter half of step 4, use pip to install the package `nvitop`. This is a tool that shows you a concise real-time summary of GPU activity. Then, run Gemma and launch nvitop at the same time:
+
+```
+(gemma-venv) user@nid001234:/capstor/scratch/cscs/user/gemma-inference$ python ./gemma-inference.py > ./gemma-output.log 2>&1 & nvitop
+```
+
+Note the use of bash `> ./gemma-output.log 2>&1` to hide any output from Python. Note also the use of the single ampersand `'&'` which backgrounds the first command and runs `nvitop` on top.
+
+After a moment, you will see your Python script spawn on all four GPUs, after which the GPU activity will increase a bit and then go back to idle. At this point, you can hit `q` to quite nvitop and you will find the output of your Python script in `./gemma-output.log`.
+
+### Collaborating in Git
+
+In order to track and exchange your progress with colleagues, it is recommended to store the EDF, Dockerfile and your application code alongside in a Git repository in a directory on `$SCRATCH` and share it with colleagues.
```

Comment:
> directory on `$SCRATCH` and share it with colleagues.

Is this a good suggestion with the cleanup policy in place? `$PROJECT`? Not sure what's the best place to share... just recommend a git repo without mentioning where to store it?